src/blueblock/Locator.java

```diff
@@ -51,55 +43,46 @@ public static Human GetHuman(int x, int y) {
 	}
 
 	public static void MovePlayer(Human h, int player, String direction) {
-		int pos1 = h.GetY();
-		int pos2 = h.GetX();
-		int Y0 = h.GetY() - 1;
-		if (Y0 < 0)
-			Y0 = 0;
-		int YG = h.GetY() + 1;
-		if (YG >= Main.FieldRows)
-			YG = h.GetY();
-		int X0 = h.GetX() - 1;
-		if (X0 < 0)
-			X0 = 0;
-		int XG = h.GetX() + 1;
-		if (XG >= Main.FieldColumns)
-			XG = h.GetX();
+		int pos1 = h.GetX();
+		int pos2 = h.GetY();
 		backgroundColor(pos1, pos2);
+
+		// TODO: x and y are somehow swapped?
```

Comment:
If you have enough time, you could eventuelly fix this.
Else we merge it and i will try.